pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -817,13 +817,14 @@ var updateHostsFile = func(entry string) error {
 	if err != nil {
 		return fmt.Errorf("failed opening file: %s", err)
 	}
-	defer file.Close()
 
 	_, err = file.WriteString(entry)
 	if err != nil {
+		file.Close()
 		return fmt.Errorf("failed writing to file: %s", err)
 	}
-	return nil
+
+	return file.Close()
```

Comment:
I prefer you follow the same convention everywhere and use the helper functions.

Reply:
Done